fitting/fitScanData.py

```diff
@@ -277,7 +277,10 @@ def fit(self, debug=False):
 
                     # Provide an initial guess
                     init_guess_p0 = self.calDAC2Q_m[vfat]*(8+stepN*8)+self.calDAC2Q_b[vfat]
-                    init_guess_p1 = abs(self.calDAC2Q_m[vfat]*rand)  #self.calDAC2Q_m[vfat] might be negative (e.g. VFAT3 case)
+                    if self.isVFAT3:
+                        init_guess_p1 = self.calDAC2Q_m[vfat]*(256)+self.calDAC2Q_b[vfat]+abs(self.calDAC2Q_m[vfat]*rand)  #self.calDAC2Q_m[vfat] might be negative (e.g. VFAT3 case)
+		    else:
+			init_guess_p1 = abs(self.calDAC2Q_m[vfat]*rand)
```

Comment:
Is the `else` in line 282 supposed to be paired with the `if` at line 280?  Seems you have a different indentation here.  Perhaps your editor is inserting `\t` characters when pressing the `tab` key instead of inserting four `\s` characters (e.g. space).

Reply:
thanks, it certainly is. i'll reformat